mne_bids/meg_bids.py

```diff
@@ -246,6 +275,8 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         The path of the BIDS compatible folder
     session_id : str | None
         The session name in BIDS compatible format.
+    acquisition : str | None
+        The acquisition data identifier
```

Comment:
what does this correspond to?

Reply:
It is one of the parameters that can be specified for the files (from the BIDS specification).
`make_bids_filename` was able to take an acquisition value as an argument, but the actual raw_to_bids function had no way of having one specified.